war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -682,53 +682,23 @@ function preventInputEe(event) {
 }
 
 /**
- * Wraps a <button> into YUI button.
+ * DEPRECATED: Wraps a <button> into YUI button.
+ *
+ * @deprecated Add the event listener to the button rather than call this method
```

Comment:
👍 used in a few plugins unfortunately: https://github.com/search?q=org%3Ajenkinsci+makeButton&type=code